### PR TITLE
USHIFT-1423: Only test upgrades on metal

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

File: ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh [hunk 1]
```diff
@@ -7,12 +7,6 @@ IP_ADDRESS="$(cat "${SHARED_DIR}"/public_address)"
 HOST_USER="$(cat "${SHARED_DIR}"/ssh_user)"
 INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
 
-# May need to update openshift-microshift-infra-aws-ec2-commands.sh to
-# open other ports if changing these settings.
-SSH_EXTERNAL_BASE_PORT=7000
-API_EXTERNAL_BASE_PORT=7100
-LB_EXTERNAL_BASE_PORT=7500
```
Comment (dhellmann) [inline ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh:14]: These variables are still being passed to the command at the end of this file when it runs the remote part of the boot process. We need to keep them until we can update that script to not need them.

Comment (pacevedom) [reply ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh:14]: This one slipped from a different PR, fixed it.

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by dhellmann** [comment]
The issue with `ostree refs` should be fixed.

/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

File: ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh [hunk 2]
```diff
@@ -46,15 +44,12 @@ cp /tmp/pull-secret "\${PULL_SECRET}"
 cd ~/microshift
 
 ./test/bin/ci_phase_iso_build.sh
-
-./scripts/image-builder/build.sh -pull_secret_file "\${PULL_SECRET}" -microshift_rpms ~/rpms -authorized_keys_file ~/.ssh/id_rsa.pub -open_firewall_ports 6443:tcp
```
Comment (dhellmann) [inline ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh:50]: I'm seeing an error on https://github.com/openshift/microshift/pull/1989 that I think will be resolved by removing this step. See https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_microshift/1989/pull-ci-openshift-microshift-main-microshift-metal-tests/1679202926033113088#1:build-log.txt%3A4889 for example

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

File: ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh (hunk 2 above)
Comment (pacevedom) [reply ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh:50]: This started happening in all builds. The issue is that there is a directory missing, but we didnt change the logic. Not sure if its worth fixing as merging this PR will make it disappear.

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pmtk** [comment]
/lgtm
/hold

unhold at your convenience

**Pull request merged**